authutils/auth_service.go

```diff
@@ -83,6 +83,19 @@ func CompareWithLatestBlockNumber(blockNumberPassed *big.Int) error {
 	return nil
 }
 
+//Check if the block number ( date on which the token was issued is not more than 1 month)
+func CheckIfTokenHasExpired(blockNumberPassed *big.Int) error {
```

Comment:
I would rename `blockNumberPassed` to `expirationBlock`

Reply:
Done

Reply:
Excuse me but I don't see the change? May be you didn't push it?

Reply:
Apologies, I missed changing the name of the parameter